grin/src/Test/ExtendedSyntax/Test.hs

```diff
@@ -0,0 +1,941 @@
+{-# LANGUAGE DeriveGeneric, LambdaCase, TypeApplications, StandaloneDeriving, RankNTypes #-}
+{-# LANGUAGE QuasiQuotes, ViewPatterns, OverloadedStrings #-}
+module Test.ExtendedSyntax.Test where
+
+import Prelude hiding (GT)
+
+import Control.Applicative
+import Control.Monad
+import Control.Monad.Extra (loopM)
+import Control.Monad.Logic
+import Control.Monad.Trans (lift)
+import Control.Monad.Identity
+import Control.Monad.Trans.Maybe
+import Control.Monad.Trans.State -- .Plus
+import Control.Monad.Trans.Reader
+import qualified Control.Monad.State.Class as CMS
+import qualified Control.Monad.Reader.Class as CMR
+import Data.Bifunctor
+import Data.Functor.Infix
+import Data.Functor.Foldable
+import Data.List ((\\))
+import Data.Maybe (fromJust, maybeToList)
+import Data.Semigroup
+import Data.String (fromString)
+import qualified Data.Text as Text
+import GHC.Generics
+import Grin.Grin hiding (Def)
+import qualified Grin.Grin as Grin
+import qualified Grin.TypeEnvDefs as Grin
+import qualified Grin.PrimOpsPrelude as PrimOps'
+-- import qualified Test.PrimOps as PrimOps
```

Comment:
Please remove the commented out dead code. It makes the code look abandoned.

Reply:
Done.